client/src/components/Nav Menu/NavMenu.css

```diff
@@ -0,0 +1,91 @@
+.nav-container {
+    background-color: #FAE5DC;
+    display: flex;
+    flex-flow: column;
+    justify-content: flex-start;
+    align-content: space-around;
+    align-items: center;
+    height: 100vh;
+    width: 100vw;
+    margin: 0 auto;
+}
+
+.nav-container nav {
+    text-align: center;
+    padding-top: 4rem;
+
+}
+
+.nav-container nav ul {
+    list-style: none;
+     margin: 0;
+     padding: 0;
+}
+
+.nav-container nav ul li h3 {
+    font-family: 'Raleway', sans-serif;
+    font-style: normal;
+    font-weight: 300;
+    font-size: 25px;
+    line-height: 29px;
+    color: #000000;
+    text-transform: uppercase;
+}
+
+span.dots {
+    display: none;
+    padding: 0 2em;
+    font-size: 8px;
+}
+
+@media only screen and (min-width: 735px) {
+
+    .nav-container {
+        flex-flow: row;
+        justify-content: space-evenly;
+        align-items: center;
+        margin: auto;
+        height: auto;
+        width: 100%;
+    }
+
+    .nav-container nav {
+        padding: 5px;
+        display: flex;
+        justify-content: space-evenly;
+    }
+
+    .nav-container nav ul {
+         display: flex;
+         flex-flow: row;
+         list-style: none;
+         margin: 0;
+         padding: 0;
+    }
+
+    .nav-container nav ul li h3 {
```

Comment:
Is there a way to use a shorter, more specific selector here? Are there classes you could use?